src/columnsTypes/types/Label/labelPopup.js

```diff
@@ -83,6 +83,7 @@ class TableLabelsMenu {
   onOpen() {
     ReactDOM.render(
       <LabelsChooser
+        handleLabelChoose={this.updateMenu(this.view)}
```

Comment:
Strange, how it worked till now?

Reply:
i think the dispatch happened only on close, so it looked like it was working (css) but wasn't really colab